app/lib/common/widgets/attachments/post_attachment_selection.dart

```diff
@@ -102,7 +102,8 @@ class _PostAttachmentSelectionState
       for (var selected in widget.attachments) {
         final type = selected.type;
         final file = selected.file;
-        final mimeType = lookupMimeType(file.path)!;
+        final mimeType = lookupMimeType(file.path);
+        if (mimeType == null) continue;
```

Comment:
Does this mean we're bypassing the error state of ```try catch``` flow? the reason for putting bang operator is to make it throw nullable error as not being able to detect mimetype is a bug.

Reply:
agreed.  though the `!` isn't the best case here, I'd say we want to show an error to the user:

```suggestion
        if (mimeType == null) throw L10.of(context).failedToDetectMimeType;
```

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1580/commits/a58331ae3abd41849b94746441cb3a8e7ae455f7